### DISCUSSION TITLE
Video Crashing

### DISCUSSION THREAD

**Discussion by E-Mello** [elementary/browser#79] [General]
I just installed elementary and when I watch a video in Firefox, it freezes the video, the voice continues, however, the image keeps freezing

Yours sincerely,
Mello.

**Comment by danirabbit** [comment]
Hey thanks for your report! Unfortunately we're not able to track or resolve issues with 3rd party apps. Please file an issue report with Mozilla regarding Firefox behavior. Thanks!

**Reply by E-Mello** [reply]
Okay, thanks for the feedback, I thought the interaction between the operating system and the browser would be up to you, I'll follow as you guided me
